cluster/index.js

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env node
+/**
+ * Copyright 2016 Lyteworx LLC.

```

Comment:
Right, will change this.  These are copies from our private repo, and our other fork will retain these copyright noticies.